compute_endpoint/globus_compute_endpoint/endpoint/config/config.py

```diff
@@ -232,11 +244,14 @@ def __init__(
         kwargs["multi_user"] = False
         super().__init__(**kwargs)
 
-        # gymnastics to satisfy mypy
-        if executors is None:
-            self.executors = (GlobusComputeEngine(),)
+        if executors and engine:
+            raise ValueError("Only one of `engine` or `executors` may be specified")
+        elif executors:
+            _e = tuple(e for e in executors)[:1]
+            assert len(_e) == 1
```

Comment:
this assert should never trigger, right? due to the slice in the previous line. but I do think a user-facing assert/raise would be helpful as opposed to the current behavior of silently throwing away extra executors

Reply:
Cool; I'll push that shortly.